specification/implementation/considerations/index.md

```diff
@@ -34,7 +34,7 @@ The maintenance of such a living environment is the role of the DILCIS Board. Th
 
 - The DILCIS Board is responsible for establishing reasonable guidelines and quality requirements for new Content Information Type specifications, and publishing these on the Board website;
 - The Board has the responsibility and mandate to manage a registry of available Content Information Type specifications which meet the guidelines and quality requirements;
-- The Board does NOT take ownership of and have responsibility of maintaining and sustaining any Content Information Type specifications;
+- The Board does NOT take ownership of or have responsibility of maintaining and sustaining any Content Information Type specifications;
```

Comment:
"responsibility for" is better than responsibility of

Reply:
Thanks!